docs/3.0.0-beta.x/community/contributing-guide.md

```diff
@@ -0,0 +1,75 @@
+# Open Development & Community Driven
+
+Strapi is an open-source project administered by [the Strapi team](https://strapi.io/company). We appreciate your interest and efforts in contributing to Strapi.
```

Comment:
We appreciate your interest and efforts in contributing to the project. (We already have Strapi 2 times in the sentence)

Reply:
I deleted this sentence as we say it better at the end of the section.